edgexfoundry/app-functions-sdk-go
# build: Add ldflags to strip debug info from binary

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

      <link to docs PR>

## Testing Instructions
Run non-secure edgex stack from compose builder
```
make run no-secty ds-virtual
```
build app-template from this branch
Verify it builds properly
run app-template
```
./app-new-service -cp -d -o
```
Verify service bootstraps successfully and starts process events from the MessageBus
## New Dependency Instructions (If applicable)
